utils/composer-packages/src/PackageResolver.php

```diff
@@ -1,18 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Ssch\TYPO3Rector\ComposerPackages;
-
-use Ssch\TYPO3Rector\ComposerPackages\Collection\ExtensionCollection;
-use Ssch\TYPO3Rector\ComposerPackages\ValueObject\ComposerPackage;
-
-interface PackageResolver
-{
-    public function findPackage(ComposerPackage $package): ExtensionCollection;
-
-    /**
-     * @return ComposerPackage[]
-     */
-    public function findAllPackagesByType(string $type): array;
-}
```

Comment:
The contract here is only one-use. It does not allow to mock ;) nor another implementation is expected. So we can use the service directly with narrowing the scope :+1: